specification/src/main/asciidoc/platform/Security.adoc

```diff
@@ -885,19 +885,19 @@ permissions that a component needs for access.
 
 === Caller Authorization
 
-A Java EE product must enforce the access
+A Jakarta EE product must enforce the access
 control rules specified at deployment time (see
 link:#a451[See Deployment
 Requirements]”) and more fully described in the EJB and Servlet
```

Comment:
You may want to refer to Jakarta Servlet but I'm not sure

Reply:
I would just combine the two -- "..described in the Jakarta Enterprise Beans and Servlet specifications"